### PR TITLE
Feature | Display topics on the Blog page #82

### PR DESCRIPTION
**Pull request by swastik4805**
Added the feature which displays topics on each blog page.
Fixes: #82 

Screenshot from 2024-05-22 23-06-24

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "prisma": "^5.13.0"
   },
   "devDependencies": {
-    "@cloudflare/workers-types": "^4.20240403.0",
+    "@cloudflare/workers-types": "^4.20240512.0",
```
Comment (aadeshkulkarni) [inline backend/package.json:24]: Please state the reasoning behind this change. 
If it was unintentional, rollback to previous state.

File: frontend/src/components/Blog/Story.tsx [hunk 2]
```diff
@@ -42,6 +43,12 @@ const Story = () => {
 					<ReactQuill value={blog?.content} readOnly={true} theme={"bubble"} />
 				</div>
 			</div>
+			{/* {blog.tagsOnPost.map((tagWrapper) => {
+				return <div key={tagWrapper.tag.id}>
+					{tagWrapper.tag.tagName}
+				</div>
+			})} */}
```
Comment (aadeshkulkarni) [inline frontend/src/components/Blog/Story.tsx:50]: Remove comments.

File: frontend/src/components/Blog/Story.tsx [hunk 3]
```diff
@@ -42,6 +43,12 @@ const Story = () => {
...
+			<Tags></Tags>
```
Comment (aadeshkulkarni) [inline frontend/src/components/Blog/Story.tsx:51]: Can we keep it <Tags /> instead ? That's the standard React, isnt it?

File: frontend/src/components/Tags.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+    return(
+        <div className="md:flex">
+            {blog.tagsOnPost.map((tagWrapper) => {
+				return (
+                    <div className="p-2" key={tagWrapper.tag.id}> 
+                        <div className="bg-gray-200 p-3 rounded-3xl flex items-center justify-center">
+                            {tagWrapper.tag.tagName}
+                        </div>
+                    </div>
```
Comment (aadeshkulkarni) [inline frontend/src/components/Tags.tsx:18]: I would recommend that we create a `Pill` component for the above code snippet. 
The component can take the `title` as prop.
This component can then be re-used in a lot of other pages in future.

**Review by aadeshkulkarni — CHANGES_REQUESTED** [review]

File: frontend/src/components/BlogCard.tsx [hunk 5]
```diff
@@ -36,7 +39,20 @@ const BlogCard = ({ author, title, content, publishedDate, id }: BlogCardProps)
...
+							<div className="px-1" key={tagWrapper.tag.id}> 
+								<div className="bg-gray-200 p-2 flex justify-center rounded-3xl text-sm">
+									{tagWrapper.tag.tagName}
+								</div>
+							</div>
+						)
+					})}
+					</div>
+					<div className="text-gray-600 pt-1">{Math.ceil(content.length / 300)} min read</div>
+				</div>
```
Comment (aadeshkulkarni) [inline frontend/src/components/BlogCard.tsx:55]: The BlogCard is out of scope of this ticket. Please rollback the changes for this component.

File: frontend/src/components/Blog/Story.tsx (hunk 2 above)
Comment (swastik4805) [reply frontend/src/components/Blog/Story.tsx]: Removed comments

**Review by aadeshkulkarni — APPROVED** [review]

**Pull request merged**